src/Application.cc

```diff
@@ -91,15 +91,15 @@ using namespace gui;
 Application::Application(int &_argc, char **_argv, const WindowType _type)
   : QApplication(_argc, _argv), dataPtr(new ApplicationPrivate)
 {
-  igndbg << "Initializing application." << std::endl;
+  gzdbg << "Initializing application." << std::endl;
```

Comment:
Oops, these can't go in yet

Reply:
Oops

Reply:
Force pushed and now this commit is :boom: